README.md

```diff
@@ -200,13 +200,15 @@ To deploy to staging:
 
 ```
 cf target -s project-monitor-staging -o IAD
+cf create-service cleardb boost rails-mysql
```

Comment:
We don't want the paid services to be part of the setup instructions. It should be user's decision to pick the most appropriate service for them. On the other hand, you are right about the number of concurrent connections. If the users' don't mind occasional errors, it should be their choice. Letting the users know about these details would be a great addition to the updated readme.